docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -147,7 +140,7 @@ use openzeppelin::access::accesscontrol::interface::IAccessControl;
 External interface of AccessControl.
 
 [.contract-index]
-.{inner-src5} ID
+.{inner-src5}
```

Comment:
i think the link is broken, it results in

`https://deploy-preview-719--oz-contracts-cairo.netlify.app/contracts-cairo/0.7.0/api/access#api/introspection.adoc#ISRC5`

Reply:
This PR doesn't have SRC5, the link will work after merging it with #721

Reply:
right but the link is broken nevertheless. it shouldn't be `api/access#api/introspection.adoc`, or are you saying it is malformed only because it could not resolve the xref?

Reply:
It works on the introspection PR, right